README.md

```diff
@@ -44,11 +44,11 @@ yarn test
 ## Authors
 
  ### **Andela Titan-Dev Team**
-- [Purity Butir](https://github.com/puritybirir) Our TTL The Coach
-- [Luc Abayo ](https://github.com/abayo-luc) Our CR7 Christiano Ronaldo
-- [Yves Iraguha](https://github.com/YvesIraguha) Our Young Talent Marcus Rashford
-- [Fabrice Niomwungeri](https://github.com/niomwungeri-fabrice) Our Defender Laurent Koscielny
```

Comment:
It looks like some names are misspelled is it how it should be or it is a typo error? if so, can you please change that @abayo-luc?

Like Niomungeri and Butir,...

Reply:
niomungeri is a GitHub username.